config/src/config/consensus_config.rs

```diff
@@ -228,6 +231,8 @@ impl Default for ConsensusConfig {
                 percentile: 0.5,
                 target_block_time_ms: 250,
                 min_block_time_ms_to_activate: 100,
+                // allow at least two spreading group from reordring in a single block, to utilize paralellism
```

Comment:
What is a spreading group?

Reply:
nit:reordering

Reply:
https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-68.md